src/components/NextButton/NextButton.tsx

```diff
@@ -9,7 +11,13 @@ const StyledButton = styled(Button)`
   margin-right: 15px !important;
 `
 
-const NextButton = ({ className, children, onClick }) => (
+interface NextButtonProps {
+  className?: string
```

Comment:
Don't think className was optional in the previous setup right?

Reply:
there is this implementation in IncidentNavigation, seems it should be optional 

```<NextButton
            onClick={(e) => {
              handleSubmit(e, next, wizardStep.formAction)
            }}
            data-testid="next-button"

```

Reply:
Left it optional.